### DISCUSSION TITLE
Option for Player to Completely Remove PVP Flagging?

### DISCUSSION THREAD

**Discussion by David7c8** [chromiecraft/chromiecraft#2643] [General]
Imagine this: You're multiboxing... nevermind. 

The point is: If you're in opposite-factionland, you're screwed. AOE + Rogue = PVP!

Suggested Solution? !PVP 

You get the idea.

Also, there may be a bug/feature that allows someone to trigger it regardless of faction.
I posted this in another discussion to clear my name of any wrongdoing. :bowtie:

**Comment by 55Honey** [comment]
Hi David7c8,
The solution to this behaviour, is to report the player in a discord Ticket, who flagged you intentionally. We consider this griefing.

**Reply by David7c8** [reply]
Sadly I have not the Orc's name. I will keep record of these events from now on and report. Thank you very much!

By the way, I noticed your request of inquiry regarding multiboxing. Care to share some source code? 😮

**Reply by 55Honey** [reply]
Sorry, source code for what?

**Reply by David7c8** [reply]
If you've ever worked on your own multibox script or program. I'd love to see!

Great work on the server itself though.

**Reply by 55Honey** [reply]
https://github.com/55Honey/MaloWBotLK
is my fork of an RTS approach at multiboxing wow. Doesn't work on CC or any other server with a default ruleset of course.

**Reply by David7c8** [reply]
Thank you for this. I'm not used to this kind of setup but I have to figure it out. There looks like some really cool stuff in here!
